AB#989 /secrets endpoint (#57)
edgelesssys/marblerun.sh: content/docs/reference/coordinator.md

```

### /secrets

For setting and retrieving secrets.

This API endpoint only works when `Users` were defined in the Manifest. For more information, look up [Managing Secrets]({{< ref "docs/workflows/managing-secrets.md" >}}).

**Returns (HTTP GET)**:
{{<table "table table-striped table-bordered">}}
| Field value                 | Type   | Description                                                      |
| --------------------------- | ------ | ---------------------------------------------------------------- |
| \<SecretName\> (one or more)| map    | A map containing key-value pairs for the requested secret.       |
{{</table>}}

Each GET requests allows specifying one or more secrets in the form of a query string, where each parameter `s` specifies one secret.
A query string for the secrets `symmetric_key_shared` and `cert_shared` may look like the following:
```
s=symmetric_key_shared&s=cert_shared
```

Example for retrieving the secrets `symmetric_key_shared` and `cert_shared`:
```bash
curl --cacert marblerun.crt --cert user_certificate.crt --key user_private.key https://$MARBLERUN/secrets?s=symmetric_key_shared&s=cert_shared
```

> Setting secrets requires uploading a file containing the secrets using a POST request. For more information refer to [Managing Secrets]({{< ref "docs/workflows/managing-secrets.md" >}}).

Reviewer: ```suggestion
Setting secrets requires uploading them in JSON format using a POST request. For more information refer to [Managing Secrets]({{< ref "docs/workflows/managing-secrets.md" >}}).
```
